components/send/PriorityFeeSelection.tsx

```diff
@@ -0,0 +1,147 @@
+import { twMerge } from "tailwind-merge";
+import React from "react";
+import { SendFormData } from "@/components/screens/Send.tsx";
+import { useFormContext } from "react-hook-form";
+import useMempoolStatus from "@/hooks/useMempoolStatus.ts";
+import usePriorityFeeEstimate from "@/hooks/usePriorityFeeEstimate.ts";
+
+type PriorityFeeSelectionProps = {
+  isPriorityFeeSelectionOpen: boolean;
+  closePriorityFeeSelection: () => void;
+};
+
+export default function PriorityFeeSelection({
+  isPriorityFeeSelectionOpen,
+  closePriorityFeeSelection,
+}: PriorityFeeSelectionProps) {
+  const { setValue, watch } = useFormContext<SendFormData>();
+  const selectedPriority = watch("priority");
+  const { mempoolCongestionLevel } = useMempoolStatus();
+  const feeEstimate = usePriorityFeeEstimate();
+
+  const formatDuration = (seconds: number): string => {
+    if (seconds < 0 || isNaN(seconds)) {
+      return "<0 sec";
```

Comment:
I think it would be better to be `<1 sec`.

Reply:
You caught me vibe coding :)

e71987ca2ad73b369fc6e88fef3d70dc06725457